prow/cmd/deck/static/tidescript.js

```diff
@@ -227,6 +227,21 @@ function createPRCell(pool, prs) {
     return c;
 }
 
+function createBatchCell(pool) {
+    var td = document.createElement('td');
+    if (pool.BatchPending) {
+        var numbers = pool.BatchPending.map(p => String(p.Number));
+        var batchRef = pool.Branch + ',' + numbers.join(',');
+        var href = '/?repo=' + encodeURIComponent(pool.Org + '/' + pool.Repo) +
+            '&type=batch&pull=' + encodeURIComponent(batchRef);
+        var link = document.createElement('a');
+        link.href = href;
+        link.appendChild(document.createTextNode(numbers.join(' ')));
+        td.appendChild(link, td.firstChild);
```

Comment:
What does the second argument do? I can't find a reference for this version of `appendChild`.